reaktive/src/commonMain/kotlin/com/badoo/reaktive/utils/Utils.kt

```diff
@@ -6,6 +6,8 @@ import com.badoo.reaktive.utils.lock.synchronized
 import kotlin.properties.ReadWriteProperty
 import kotlin.reflect.KProperty
 
+internal object Uninitialized
```

Comment:
It would be nice if you replace `dummyValue` in `CombineLatest.kt` with `Unitialized`

Reply:
Will send another PR for that one ;)

Reply:
#35